BraveAlerterConfigurator.js

```diff
@@ -14,6 +14,8 @@ class BraveAlerterConfigurator {
       this.getAlertSessionByPhoneNumber.bind(this),
       this.alertSessionChangedCallback.bind(this),
       this.getLocationByAlertApiKey.bind(this),
+      this.getHistoricAlertsByAlertApiKey.bind(this),
+      () => {},
```

Comment:
This is because the `brave-alert-lib` upgrade includes Dave's code for the notification count endpoint. This empty function will eventually be filled in by him for that.